Source/ARTPushActivationStateMachine.m

```diff
@@ -226,6 +226,7 @@ - (void)deviceUpdateRegistration:(ARTErrorInfo *)error {
     NSMutableURLRequest *request = [NSMutableURLRequest requestWithURL:[[NSURL URLWithString:@"/push/deviceRegistrations"] URLByAppendingPathComponent:local.id]];
     request.HTTPMethod = @"PATCH";
     request.HTTPBody = [[_rest defaultEncoder] encode:@{
+        @"clientId": local.clientId,
```

Comment:
@paddybyers Is this necessary for the new release?

Reply:
Removed that line because ClientId should be immutable after a successful device registration.